src/main/java/nextstep/subway/entity/Sections.java

```diff
@@ -15,14 +15,64 @@ public class Sections {
 	@OneToMany(mappedBy = "line", cascade = CascadeType.PERSIST, orphanRemoval = true)
 	private List<Section> sections;
 
-	protected Sections() {
+	private Long startStationId;
+
+	private Long endStationId;
+
+	private int distance;
+
+	public Sections() {
+	}
```

Comment:
jpa를 위한 생성자이니 접근 범위는 낮춰도 좋을 것 같아요
```suggestion
	protected Sections() {
	}
```